### DISCUSSION TITLE
Is it safe if i run it locally and give it personally identifiable data?

### DISCUSSION THREAD

**Discussion by ygivenx** [open-webui/open-webui#1857] [Q&A]
A more general question is how can I prove to my security team that this software is not sending any data outside?

**Comment by justinh-rahb** [comment]
Put it on an isolated network and/or watch outgoing traffic coming from the container. We aim to ensure there's nothing that _depends_ on internet access if the user wishes to lock things down, that said there are some legitimate external calls that may occur (updating internal models for embedding/TTS and other things of that nature). If you find anything calling home that shouldn't be, please file an issue report so we can look into it. We have a long and documented history of taking such issues seriously.

**Comment by pkeffect** [comment]
https://www.wireshark.org/ is a must have for network analyzing.
